README.md

```diff
@@ -72,8 +72,14 @@ Follow these steps to set up and run the GitHub App, and get the webhook server
      - `GITHUB_PRIVATE_KEY_PATH`: relative path to .pem file (e.g., 'private-key.pem')
      - `GITHUB_APP_USER_NAME`: the 'name' of your GitHub app, followed by '[bot]'
      - `GIT_COMMIT_AUTHOR_NAME` and `GIT_COMMIT_AUTHOR_EMAIL`: your details which will be used in Aider's commits. You can find your public-facing GitHub email address in the [GitHub settings](https://github.com/settings/emails).
+     - `CONVENTIONS_FILE_PATH`: (Optional) Set this to the path of a Markdown file within your repository that contains project-specific coding conventions. If set, Aider will use these conventions when making coding changes.
 
-5. **Install the App:**
+5. **Set up project-specific conventions (Optional):**
```

Comment:
@Aiderbot when we say **Set up project-specific conventions (Optional):**, can we please insert a note to say you can look at the CONVENTIONS.md file in this repo as an example?